lib/cortex/snippets/webpage.rb

```diff
@@ -2,109 +2,123 @@ module Cortex
   module Snippets
     class Webpage
       def initialize(cortex_client, url)
-        @webpage = cortex_client.webpages.get_feed(url).contents
+        @webpage = cortex_client.webpages.get_feed(url)
+        @contents = @webpage.contents
+        @page_cache_key = @webpage.status == 200 ? "view:#{url}@#{@contents.updated_at}" : "view:#{url}"
+      end
+
+      def response
```

Comment:
I see the getter in question here - let's have the getter match the name of the instance variable. I'd recommend we change the instance variable to `@response` and use `attr_reader` to setup the getter.

Reply:
got it